gms/api/src/main/pegasus/com/linkedin/ml/MLFeature.pdl

```diff
@@ -0,0 +1,41 @@
+namespace com.linkedin.ml
+
+import com.linkedin.common.ChangeAuditStamps
+import com.linkedin.common.MLFeatureUrn
+import com.linkedin.common.VersionTag
+import com.linkedin.common.Ownership
+import com.linkedin.common.InstitutionalMemory
+import com.linkedin.common.Status
+import com.linkedin.common.Deprecation
+import com.linkedin.ml.metadata.MLFeatureProperties
+
+/**
+ * MLFeature spec. for a feature store. A collection of MLFeature metadata schema that can evolve over time.
+ */
+record MLFeature includes MLFeatureKey, ChangeAuditStamps, VersionTag {
```

Comment:
similar comment as the one made for `MLModel`.

Reply:
`MLFeatureProperties` aspect does not have `version` field like `MLModelProperties` does.
Do you mean to add `version` field in the `MLFeatureProperties` and remove it here?

Reply:
Sounds good.